gui-apps/wf-recorder/wf-recorder-9999.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Gentoo Authors
```

Comment:
Adding this file should happen in a separate commit. So 1 PR, 2 commits. But please sync this with requested changes to 0.2.1.

Reply:
done